### PR TITLE
Adding CREST-2.9.eb

### PR DESCRIPTION
**Pull request by dplacencia**
CREST software is a dependency of the xtb software

### CODE REVIEW

File: easybuild/easyconfigs/c/CREST/CREST-2.9.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+easyblock = 'Binary'
+
+name = 'CREST'
+version = '2.9'
```
Comment (smoors) [inline easybuild/easyconfigs/c/CREST/CREST-2.9.eb]: @dplacencia please add the release date as local var:
```
local_verdate = '200219'
```

**Comment by dplacencia** [comment]
> @dplacencia do you have a specific reason why you're using a pre-release version instead of a release?
Hi, I am working together with the developers of xTB to publish a release soon. At this time the latest version of this software is available only in pre-release. Could this change be postponed?

**Comment by awvwgk** [comment]
@dplacencia Thanks for your effort, we are always happy to make our programs widely available. *But* since this is a binary redistribution of the `crest` program, which is not governed under an open source license right now, so you should ask the authors of the program first.

I guess there will be no problem with adding `crest` here, but I am not the one who can actually give you allowance for this. I kindly ask you suspend this PR for the moment and contact the authors of `crest` first.

Also pinging @pprcht since he is one of the authors of `crest`.

**Comment by dplacencia** [comment]
closing pull request momentarily until authorization is obtained

**Pull request closed without merging**